apps/core/src/hooks/useTokenPrice.ts

```diff
@@ -7,24 +7,36 @@ import BigNumber from 'bignumber.js';
 
 import { useAppsBackend } from './useAppsBackend';
 import { useCoinMetadata } from './useFormatCoin';
+import { Feature, FiatTokenName } from '../enums';
+import { COIN_TYPE_TO_FIAT_TOKEN_NAME } from '../constants/coinTypeToFiatTokenName.constants';
```

Comment:
add to `apps/core/src/constants/index.ts` and fix this import :pray: 
```suggestion
import { COIN_TYPE_TO_FIAT_TOKEN_NAME } from '../constants';
```